docs/articles/documentation/reference/test-api/testcontroller/expect/README.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: docs
+title: t.expect Method
+permalink: /documentation/reference/test-api/testcontroller/expect/README.html
+---
+# t.expect() Method
+
+This method accepts the actual value. You can pass a value, a [Selector's DOM node state property](../../../../guides/basic-guides/select-page-elements.md#define-assertion-actual-value)
+or a [client function](../../../../guides/basic-guides/obtain-data-from-the-client.md) promise.
+TestCafe automatically waits for node state properties to obtain a value and for client functions to execute.
+See [Smart Assertion Query Mechanism](../../../../guides/basic-guides/assert.md#smart-assertion-query-mechanism) for details.
```

Comment:
I think we should say that this method should be followed by assertion methods (and even link them all, like in the guide).

We should also say about the value type (as we do in the template).

And add an example.